src/components/SearchInput/SearchInput.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import SearchInput from '@/components/SearchInput';
+
+export default {
+  title: 'Search Input',
+  component: SearchInput
+} as Meta;
+
+const Template = args => <SearchInput {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  size: 'large',
+  placeholder: 'Search Anything'
+};
+
+export const SearchWithValue = Template.bind({});
```

Comment:
Maybe rename it to "DefaultWithValue" since in no other you have the "Search" in the beginning?